Src/IronPython/Runtime/Bytes.cs

```diff
@@ -16,6 +16,7 @@
 
 using IronPython.Runtime.Operations;
 using IronPython.Runtime.Types;
+using System.Text;
```

Comment:
nit: Move to up to the "using System" group?

Reply:
OK